azurelinuxagent/common/utils/networkutil.py

```diff
@@ -179,34 +167,30 @@ def get_accept_tcp_rule(command, destination, firewalld_command="", wait=""):
             cmd = AddFirewallRules.__get_firewalld_base_command(firewalld_command)
         else:
             cmd = AddFirewallRules.__get_iptables_base_command(wait)
-        cmd = cmd + ['-t', 'security', command, 'OUTPUT', '-d', destination, '-p', 'tcp', '--destination-port', '53', '-j', 'ACCEPT']
+        cmd = cmd + ['-t', 'security', command, 'OUTPUT', '-d', destination, '-p', 'tcp', '--destination-port', '53',
+                     '-j', 'ACCEPT']
         return cmd
 
     @staticmethod
-    def get_iptables_accept_command(wait, command, destination, owner_uid):
-        cmd = AddFirewallRules.__get_iptables_base_command()
-        cmd.extend(AddFirewallRules.__get_common_accept_command_params(wait, command, destination, owner_uid))
-        return cmd
+    def get_accept_command(command, destination, owner_uid, firewalld_command="", wait=""):
+        if firewalld_command != "":
```

Comment:
Thanks for all the refactoring and the effort into making this code cleaner. Just 1 more request, I noticed this `if-else` condition is common for all 3 of our rules, maybe consider making this a common function too?